MusicPlayer/MusicPlayer/ViewController.swift

```diff
@@ -13,25 +15,126 @@ final class ViewController: UIViewController {
     @IBOutlet private weak var songImage: UIImageView!
     @IBOutlet private weak var songLabel: UILabel!
     @IBOutlet private weak var artistLabel: UILabel!
-    private var isPlay = true
-    var selectedSong : Song?
+    @IBOutlet private weak var prevButton: UIButton!
+    @IBOutlet private weak var nextButton: UIButton!
+    @IBOutlet private weak var slider: UISlider!
+    
+    var songs: [Song] = []
+    var position: Int = 0
```

Comment:
```suggestion
    var position = 0
```
Xem lại Swift coding convention nha Bách

Reply:
oke mình sửa lại rồi nha